### PR TITLE
Provide "Migrating our container images to Github Container Registry" blog post

### PR DESCRIPTION
**Pull request by tomkerkhove**
Signed-off-by: Tom Kerkhove <kerkhove.tom@gmail.com>

Relates to https://github.com/kedacore/governance/issues/16

### CODE REVIEW

File: content/blog/migrating-to-github-container-registry.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
++++
+title = "Migrating our container images to Github Container Registry"
+date = 2021-03-31
```
Comment (tomkerkhove) [inline content/blog/migrating-to-github-container-registry.md:3]: Happy to change this to what we want, based on how fast this gets approved.

File: content/blog/migrating-to-github-container-registry.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+**If you are using one of our deployment options, end-users are not be impacted.**
+
+Since v2.2, we are using GitHub Container Registry by default and you are good to go.
+
+If you are using your own deployment mechanism, then you will have to pull the container images from GitHub Container Registry instead.
+
+## Join the conversation
+
+Do you have questions or remarks? Feel free to join the conversation on [GitHub Discussions](TODO-Open-Before-Merge).
```
Comment (tomkerkhove) [inline content/blog/migrating-to-github-container-registry.md:67]: Happy to create it already, if ok, but would include some of the content here.

**Review by zroubalik — APPROVED** [review]
Looking good Tom!

File: content/blog/migrating-to-github-container-registry.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
++++
+title = "Migrating our container images to Github Container Registry"
```
Comment (tomkerkhove) [inline content/blog/migrating-to-github-container-registry.md:2]:
```suggestion
title = "Migrating our container images to GitHub Container Registry"
```

**Pull request merged**